string data
geometry_msgs/PoseStamped pose
